docs/source/changelog.rst

```diff
@@ -1,13 +1,19 @@
 =========
 Changelog
 =========
-**Future Release**
+
+**v0.2.0** April 23, 2020
     * Changes
         * Rename LabelTimes.name to LabelTimes.label_name (:pr:`126`)
+        * Support keywords arguments in Pandas methods. (:pr:`121`)
+    * Documentation Changes
+        * Improved data download in Predict Next Purchase (:pr:`76`)
+    * Testing Changes
+        * Removed tests that use Python 3.5 in CirlceCI (:pr:`128`)
```

Comment:
I think it would be good to mention in the changelog that Compose is dropping Python 3.5 support with this release.

Maybe we could move this line about no longer testing 3.5 to the Changes section and make it about dropping Python 3.5